ui/lib/hotkeys.js

```diff
@@ -11,6 +11,14 @@ export const initializeHotkeys = () => {
     push(path.designSystemGuide());
   });
 
+  // temporary hotkey to open user registration
```

Comment:
```suggestion
  // TODO(merle): Remove temporary hotkey to open user registration
```

Make it more actionable. And helps audits for outstanding cleanups.